pyquil/paulis.py

```diff
@@ -389,7 +389,7 @@ def from_compact_str(cls, str_pauli_term):
 
         return op
 
-    def pauli_string(self, qubits=None):
+    def pauli_string(self, qubits=None) -> str:
```

Comment:
```suggestion
    def pauli_string(self, qubits: Optional[Iterable[int]] = None) -> str:
```

Might have to import `Iterable`. Also, update the docstring to say `Iterable` instead of `list` where appropriate.